### PR TITLE
[Update] Android dependencies

### PR DESCRIPTION
**Pull request by m1ga**
Update Crashlytics dependencies.
Issue found (see 2nd post)
```
var crash = require("ti.crashlytics");
crash.crash()
```

### CODE REVIEW

**Comment by hansemannn** [comment]
@m1ga Would it be possible to place this in onCreate like suggested by him? I think there is a override method for that!

**Comment by m1ga** [comment]
@hansemannn I'll check it again :+1: I've tried `onAppCreate` before but it could be that it was a configuration issue at the time I've tried that. Now I know it should work so let me try it again. Of course that would be easier to use

**Comment by m1ga** [comment]
`onAppCreate()` worked! Removed the `init()` part again. So now its a simple module update for existing users.

File: android/src/ti/crashlytics/TitaniumCrashlyticsModule.java [hunk 1]
```diff
@@ -50,4 +44,17 @@ public void setCrashlyticsCollectionEnabled(boolean crashlyticsCollectionEnabled
...
+		FirebaseCrashlytics.getInstance().recordException(new Exception());
+	}
+
+
+	// @Override
+	// protected void initActivity(Activity activity) {
+	// 	super.initActivity(activity);
+	// 	Log.i("-----", "App init activity");
+	// 	FirebaseCrashlytics.getInstance().recordException(new Exception());
+	// }
```
Comment (hansemannn) [inline android/src/ti/crashlytics/TitaniumCrashlyticsModule.java]: Can you remove the empty lines and commented out code? Then it should be good to go

Comment (m1ga) [reply android/src/ti/crashlytics/TitaniumCrashlyticsModule.java]: they are removed already

**Comment by hansemannn** [comment]
Thanks @m1ga, this is great! Can you draft a new release?

**Pull request merged**